src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import static core.basesyntax.ColorSupplier.RANDOM;
+
+public class FigureSupplier {
+    private static final int FIGURE_COUNT = 5;
+    private static final int MIN_LENGTH = 2;
+    private static final int MAX_LENGTH = 10;
+
+    public Figure getRandomFigure() {
+        int ran = RANDOM.nextInt(FIGURE_COUNT);
+        int a = RANDOM.nextInt(MAX_LENGTH + 1);
+        int b = RANDOM.nextInt(MAX_LENGTH + 1);
+        int c = RANDOM.nextInt(MAX_LENGTH + 1);
+        switch (ran) {
+            case (0):
+                return new Square(a);

```

Comment:
In all cases you should pass random size and random color, so create and use `ColorSupplier`, For example like this (You also need random here): 

```java
private static final ColorSupplier colorSupplier = new ColorSupplier();
private static final Random random = new Random();
private static final int FIGURE_COUNT = 5;
private static final int MIN_LENGTH = 2;
private static final int MAX_LENGTH = 10;
```